monailabel/monailabel_pathology_nuclei_segmentation_QuPath.ipynb

```diff
@@ -106,21 +106,7 @@
     "    \n",
     "```bash\n",
     "--conf auth_token <Github Auth Token>\n",
```

Comment:
Do we still need to say the `auth_token` logic?
CC @yiheng-wang-nv

Reply:
Users might face rate limit issues when using it in MONAI label. 
It's better here to remind users that they can add token as conf argument.

Reply:
I mean this is a common issue of `bundle download` API, not related to MONAI Label specific logic.
We should highlight it somewhere in the `download` doc-string, and you just need to put a link here for advanced options.
@yiheng-wang-nv Could you please improve the doc-string of `download` to help users avoid this issue?

Thanks.

Reply:
That also work. Note here, MONAI Label has a way for users to provide auth_token when starting monai label server. 
This is Label specific logic, that can pass auth token to `download`. 
Let me know if we'd better remove this here.

Reply:
Hi @tangy5 ,

OK, if so, to avoid too big change in MONAI Label, let's keep it here, but I would suggest to add more description to make it more clear.
How about this: `bundle.download` can support multiple sources, if downloading the bundle from Github links, user may need to provide the Github `auth_token` to avoid download limitation."
CC @yiheng-wang-nv 
What do you think?

Thanks.

Reply:
Hi @tangy5 , I think in MONAI label side, `auth_token` is not used for `download` API as there is no related arg.
see:
https://github.com/Project-MONAI/MONAILabel/blob/3421b4a7034514707b42efa741fa1e8903a5ba33/monailabel/utils/others/generic.py#L337

Reply:
`auth_token` is only used for requesting github release related information, and used in these functions:
```
get_all_bundles_list
get_bundle_versions
get_bundle_info
```
Thus the only place can be utilized for `download` is when we need to get latest version from github source.

Reply:
to make it more clear, let me submit a quick draft PR about it